src/porepy/fracs/fracture_importer.py

```diff
@@ -24,6 +19,7 @@ def network_3d_from_csv(file_name, has_domain=True, tol=1e-4):
         file_name: name of the file
         has_domain: if the first line in the csv file specify the domain
         tol: (optional) tolerance for the methods
```

Comment:
Could we be more specific?

Reply:
I wish. But I can add some minor comments.